cs3api4lab/tests/test_cs3apismanager.py

```diff
@@ -26,59 +26,58 @@ def test_get_text_file(self):
         file_id = "/test_get_text_file.txt"
         message = "Lorem ipsum dolor sit amet..."
         self.file_api.write_file(file_id, message, self.endpoint)
```

Comment:
this should be inside the try catch perhaps?

Reply:
but if write_file fails before try block, the file is not created anyway?

Reply:
other things may happen inside the write_file method after the file is created, for example a problem with locks